docs/tutorials/hello-nais.md

```diff
@@ -122,6 +122,10 @@ Add the following content to the file, and insert the appropriate values in the
       push:
         branches:
           - main
+    permissions:
+      packages: 'write'
+      contents: 'read'
+      id-token: 'write'
```

Comment:
Du skal ikke trenge `write` til `packages` eller `id-token`, og `contents` er allerede satt i jobben som kjører.

Reply:
Skjønner. 

Ref https://nav-it.slack.com/archives/C6VFKJ3EJ/p1718088930329409 , så måtte jeg gjøre det for å få kjørt deploy-jobben til GAR (ble pekt repoet i https://github.com/navikt/tsm-event-source-test/blob/main/.github/workflows/deploy.yaml av en kollega som hadde fått det til)